rhel/SPECS/grafana.spec

```diff
@@ -54,15 +55,17 @@ mv vendor/gopkg.in   _build/src/
 
 mkdir -p ./_build/src/github.com/grafana
 ln -s $(pwd) ./_build/src/github.com/grafana/grafana
-export GOPATH=$(pwd)/_build:%{gopath}
+export GOPATH="$(pwd)/_build"
 
 export LDFLAGS="$LDFLAGS -X main.version=%{version} -X main.commit=%{shortcommit} -X main.buildstamp=$(date '+%s') "
 %gobuild -o ./bin/grafana-server ./pkg/cmd/grafana-server
 %gobuild -o ./bin/grafana-cli ./pkg/cmd/grafana-cli
-/usr/bin/node --max-old-space-size=4500 /usr/bin/grunt --verbose --pkgVer=%{version} release
+yarn install
+/usr/bin/node --max-old-space-size=4500 /usr/bin/npm --verbose run build
```

Comment:
nit: I would suggest to not run node explicitly with --max-old-space-size, since package.json features just `npm run build`. This should be enough.